### DISCUSSION TITLE
Is there a way to import recurrence exceptions?

### DISCUSSION THREAD

**Discussion by andypullen** [solspace/craft-calendar#206] [Questions & Answers] [answered]
It doesn't look like there is any way to import recurring event exceptions. Am I missing something regarding this? I'd love to be able to feed in an event's EXDATE data and have the plugin slurp that up.

**Comment by kjmartens** [comment, marked as answer]
This currently isn't possible at this time, but it's on our list for future consideration.

**Reply by andypullen** [reply]
Thank you
